src/addIntegrationIdToConfigs.js

```diff
@@ -0,0 +1,110 @@
+const {
+  get,
+  isEmpty,
+  flow,
+  first,
+  split,
+  set,
+  parseInt,
+  last,
+  join,
+  includes,
+  assign,
+  omit,
+  findIndex
+} = require('lodash/fp');
+const { parseErrorToReadableJSON } = require('./dataTransformations');
+const {
+  createOrUpdateFile,
+  getExistingFile,
+  parseFileContent
+} = require('./octokitHelpers');
+const { v5: uuidv5 } = require('uuid');
```

Comment:
@penwoodjon Do we want to use a v5 uuid or v1?  If we use v5 then the uuid is consistent in that it is tied to the repo name and not truly random.  If for some reason we had to shuffle repos around then you could end up with an inadvertent collision.